lib/formatters/__tests__/tapFormatter.test.mjs

```diff
@@ -32,7 +29,7 @@ describe('tapFormatter', () => {
 
 		expect(output).toBe(
 			`
-TAP version 13
+TAP version 14
 1..1
 ok 1 - path/to/file.css
 `.trimStart(),
```

Comment:
[suggest] For consistency, the following seems a bit better. But this is super minor. I'm okay with leaving it as-is, too.

```js
expect(output).toBe(`TAP version 14
1..1
ok 1 - path/to/file.css
`);
```